lib/interactive_viewer.ml

```diff
@@ -4,11 +4,18 @@ open Lwd_infix
 
 let operation_info z_patches : ui Lwd.t =
   let$ z = Lwd.get z_patches in
+  let p = Zipper.get_focus z in
+  let num_hunks = List.length p.Patch.hunks in
+  let hunk_text = match num_hunks with
```

Comment:
You need to run autoformatting with `dune build @fmt --auto-promote` (or via the "format on save" feature of the ocaml vscode plugin). You need to install `ocamlformat.0.26.1` with Opam to do that.

Reply:
Hi @Julow 
I have removed the unnecessary space after the string in the status bar and also formatted the code with the `dune build @fmt --auto-promote` command.

Feels good to learn this autoformatting.

Kindly review.